### ISSUE TITLE
Why remove the RTMP support ?

### ISSUE THREAD

**Issue by jpxiong** [Kickflip/kickflip-android-sdk#33]
I noticed that there was RTMP support. But a patch which remove RTMP had been merged. Could you tell me why ?

**Comment by OnlyInAmerica** [comment]
Please see https://github.com/Kickflip/kickflip-android-sdk/issues/30

We don't think RTMP serves the mobile broadcasting use case as well as HLS, and so we don't want to dedicate our limited resources towards supporting it.

Would you mind letting me know why you need RTMP over HLS?

**Comment by jpxiong** [comment]
@OnlyInAmerica I think the latency of RTMP less than HLS. My project is about live streaming.

**Comment by gouravd** [comment]
I have successfully tested RTMP using KickFlip's modified SDK to our server(ngins rtmp module) and the latency can be easily reduced to 2 secs.

Though the major issues I had in re-adding the RTMP support was actually finding ffmpeg with rtmp. I found that the files in this commit - https://github.com/Kickflip/kickflip-android-sdk/tree/962aa39367d147bae54ccd094e03888833d00a95/sdk/src/main/jniLibs/armeabi worked. Anything after that did not work for rtmp streaming.

I am a windows guy and have no knowledge in Linux. Can anyone help in getting the static builds for latest ffmpeg with rtmp for android. I wish Kickflip had not removed the support :(

I wish Kickflip re-adds FFMpeg with rtmp support for both ARM and X86

**Comment by mikelambert** [comment]
@OnlyInAmerica for what it's worth, Facebook Live _only_ supports RTMP to upload video AFAICT:
https://developers.facebook.com/docs/videos/live-video/getting-started#creating

Sad to see that Kickflip removed RTMP upload support, as I'm looking for a library that does exactly that, and it looks like there really aren't many (any?) other options to do that.

**Comment by dmitry-saritasa** [comment]
Yes I'm interested in RTMP option as well - again because of facebook live stream API & RTMP marriage
